kategory/src/test/kotlin/kategory/instances/ComposedInstancesTest.kt

```diff
@@ -9,33 +9,33 @@ typealias OptionTNel = HK<OptionTKindPartial<NonEmptyListHK>, Int>
+                a.unnest().ev() == b.unnest().ev()
     }
 
-    val EQ_LKW_OPTION: Eq<HK<ComposedType<ListKWHK, OptionHK>, Int>> = object : Eq<HK<ComposedType<ListKWHK, OptionHK>, Int>> {
-        override fun eqv(a: HK<ComposedType<ListKWHK, OptionHK>, Int>, b: HK<ComposedType<ListKWHK, OptionHK>, Int>): Boolean =
-                a.lower().ev() == b.lower().ev()
+    val EQ_LKW_OPTION: Eq<HK<Nested<ListKWHK, OptionHK>, Int>> = object : Eq<HK<Nested<ListKWHK, OptionHK>, Int>> {
+        override fun eqv(a: HK<Nested<ListKWHK, OptionHK>, Int>, b: HK<Nested<ListKWHK, OptionHK>, Int>): Boolean =
+                a.unnest().ev() == b.unnest().ev()
     }
 
-    val EQ_OPTIONT_ID_NEL: Eq<HK<ComposedType<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>> =
-            object : Eq<HK<ComposedType<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>> {
-                override fun eqv(a: HK<ComposedType<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>, b: HK<ComposedType<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>): Boolean =
-                        a.lower().value().value().fold(
-                                { b.lower().value().value().isEmpty },
+    val EQ_OPTIONT_ID_NEL: Eq<HK<Nested<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>> =
+            object : Eq<HK<Nested<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>> {
+                override fun eqv(a: HK<Nested<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>, b: HK<Nested<OptionTKindPartial<IdHK>, OptionTKindPartial<NonEmptyListHK>>, Int>): Boolean =
+                        a.unnest().value().value().fold(
+                                { b.unnest().value().value().isEmpty },
                                 { optionA: OptionTNel ->
-                                    b.lower().value().value().ev().fold(
+                                    b.unnest().value().value().ev().fold(
                                             { false },
                                             { it.value() == optionA.value() })
                                 })
             }
 
-    val cf: (Int) -> HK<ComposedType<OptionHK, NonEmptyListHK>, Int> = { it.nel().some().lift() }
+    val cf: (Int) -> HK<Nested<OptionHK, NonEmptyListHK>, Int> = { it.nel().some().nest() }
 
     init {
         testLaws(FunctorLaws.laws(ComposedFunctor(Option.functor(), NonEmptyList.functor()), cf, EQ_OPTION_NEL))
-        testLaws(FunctorFilterLaws.laws(ComposedFunctorFilter(OptionT.functorFilter(Id.monad()), OptionT.functorFilter(NonEmptyList.monad())), { OptionT.pure(OptionT.pure(it, NonEmptyList.monad()), Id.monad()).lift() }, EQ_OPTIONT_ID_NEL))
+        testLaws(FunctorFilterLaws.laws(ComposedFunctorFilter(OptionT.functorFilter(Id.monad()), OptionT.functorFilter(NonEmptyList.monad())), { OptionT.pure(OptionT.pure(it, NonEmptyList.monad()), Id.monad()).nest() }, EQ_OPTIONT_ID_NEL))
```

Comment:
Should `ComposedFunctorFilter` be renamed to `NestedFunctorFilter`? Same for others that were previously under the Composed* names.